package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "videbate",
   "type": "module",
-  "version": "0",
+  "version": "1.0.0",
```

Comment:
We are still knee-deep in "zero version", but you can bump to `"0.0.1"` and as a RASAR you should :muscle: